package.json

```diff
@@ -157,7 +157,6 @@
     "peerDependencyRules": {
       "allowedVersions": {
         "eslint": "*",
-        "@typescript-eslint/eslint-plugin": "^5.6.0",
```

Comment:
The `peerDependencyRules.allowedVersions` exception on `@typescript-eslint/eslint-plugin` no longer seems to be necessary. Removing it does not introduce any new peer dependency errors on `pnpm install`.

I suspect this was needed for the `eslint-config-standard-with-typescript` dependency in the past, but a `@typescript-eslint/eslint-plugin` upgrade made it no longer necessary.